btb_phylo.py

```diff
@@ -238,15 +239,17 @@ def append_multi_fasta(s3_bucket, s3_key, outfile):
     """
     # temp directory for storing individual consensus files - deleted when function
     # returns
-    with tempfile.TemporaryDirectory() as temp_dirname:
-        consensus_filepath = os.path.join(temp_dirname, "temp.fas") 
+    consensus_filepath = consensus_path + sample['Sample'] + '.fas'
```

Comment:
could use `os.path.join()` as this will automatically put a `/` between the folder and the filename.